### ISSUE TITLE
A11y_PowerShellEditorServices_Editor_HighContrast: All the color themes of VSCode is below the expected ratio i.e 7:1

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61589]
**Environment Details:**
VSCode Version : 1.25.0
OS Version : Win10

**Additional Details:**
HighContrast,MAS1.4.3

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Plain Text" of "Status Bar"->Enter "PowerShell" in "Select Language Mode" combo box->Write the PowerShell Code in plain editor->Save the editor.
3.Navigate to Setting button of Activity Bar and select "Color Theme" option.
4.Color Themes shows number of different colors list apply all that theme on the editor.
5.Open color contrast analyser and check the color contrast ratio.
6.It shows color contrast ratio less than expected color contrast ratio.
 
**Actual:**
After applying color themes to editor it shows color contrast ratio as 3.5:1
eg. Select "QuietLight" theme from color theme list and check for the color contrast ratio.

**Expected:**
The expected colorcontrast ratio should be greater than or equals to 7:1 ratio.

**Recommendations:**
Refer below link which is repository of bug fixes code [](snippets:https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
)

**User Impact:**
User will not able to see the content of the editor.

**MAS Reference:**
MAS1.4.3- [https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D](url)

**Attachment For Reference:**
![a11y_powershelleditorservices_editor_colorcontrast_2726556](https://user-images.githubusercontent.com/39204909/47353278-7987f280-d6d9-11e8-8fbe-908a4cd1fa39.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
@AccessibilityTestingTeam-TCS I believe you are using the wrong color contrast ratio. From the [MAS 1.4.3](https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D) link you provided:
> Visible text must have a minimum luminosity contrast ratio of 4.5:1 against the background, except:
